### ISSUE TITLE
.exists() is no longer working

### ISSUE THREAD

**Issue by Le0Developer** [alisaifee/coredis#24]
```py
import asyncio

from coredis import StrictRedis


async def main():
    db = StrictRedis()
    await db.set("test", "some value")
    print(await db.exists("test"))


if __name__ == "__main__":
    asyncio.run(main())
```
Always prints `False`.

**Comment by Le0Developer** [comment]
The issue is probably `response_callback=BoolCallback(),` in the redis_command decorator (it should be an int)

**Comment by Le0Developer** [comment]
Oh ok, seems like exists takes a iterable now. `await db.exists("test")` -> `await db.exists(("test",))` isn't clean at all.

Keeping this open for the wrong response callback.

**Comment by alisaifee** [comment]
@Le0Developer yes `exists` and a bunch of other methods no longer accept var args when the arguments aren't optional. Specifically for `delete` and `exists` it feels a bit awkward I agree - but I decided to go with consistency.
